AccountService.js

```diff
@@ -5,6 +5,11 @@
 
 import axios from 'axios';
 const headers = {Accept: 'application/ld+json, application/json'};
+/**
+ * @class AccountService
```

Comment:
Are all the explicit tags needed?  Nicer to just let it pick up class and method names automatically.  I know it sometimes can't figure it out without some help.

Reply:
Force of habit. can try it with out them.
- [ ] recompile with out @method or @class jsdoc comments.

Reply:
this is actually turned out to be better classes now have short descriptions after them at the top and method are marked as instances instead of static. so implicit tags seem to do more.